project_one/index.html

```diff
@@ -0,0 +1,49 @@
+<body>
```

Comment:
Great job, you got the 5 required grading points. What about the extra credit options in the syllabus? Please specify which two you have done so you can get this 90 to a 100.